plugin/src/main/java/io/papermc/hangarpublishplugin/internal/HangarVersionPublisher.java

```diff
@@ -53,7 +53,7 @@ public void uploadVersion(final HangarPublication publication) throws IOExceptio
         final HangarVersion version = HangarVersion.fromPublication(publication);
         final List<File> files = version.files().stream()
             .filter(HangarVersion.FileData::isFile)
-            .map(fileData -> publication.getPlatforms().getByName(fileData.firstPlatform()).getJar().getAsFile().get())
+            .map(fileData -> publication.getPlatforms().getByName(fileData.firstPlatform()).getJar().getAsFile().getOrNull())
```

Comment:
This should never be empty/null after the isFile check from above, else bad data was provided